CODING_STANDARDS.md

```diff
@@ -37,14 +37,23 @@ The commit messages are typically rendered in [Markdown format](https://docs.git
 When updating your code, please make sure you perform a rebase, not a merge with the latest branch.
 Pull requests should be a simple fast-forward of the branch they are intended to land on.
 
-The correct way to rebase (if working on top of 10.3 branch):
+The correct way to rebase (if working on top of 10.6 branch):
 
 ```sh
-git fetch upstream/10.3  # This assumes upstream is github.com/MariaDB/server
-git rebase upstream/10.3
+git fetch upstream/10.6  # This assumes upstream is github.com/MariaDB/server
+git rebase upstream/10.6
 git push --force my_branch
 ```
 
+### Target branch
+
+Pull requests should be based against the correct MariaDB version.
+New features should be based against the latest MariaDB development branch.
+Bug fixes should be based against the earliest maintained branch in which the bug can be reproduced.
+
+* Current development branch: __11.4__
+* Current earliest maintained branch: __10.6__
+
```

Comment:
I doubt this should be specified in this file. You're pushing this into 11.4 - ok, it'll be updated there, but will become outdated in the all other (maintained) branches. 

If you'll PR this to the earliest branch, then there will be more cost to create the new major version -- apart from bumping VERSION we'll need to bump this text from the earliest branch. 

The same applies to obsoleting the branch.

What can be included though -- is some link to the actual branch list.
The best source I've found is https://endoflife.date/mariadb
Also I was just lucky to find [Maintainance policy](https://mariadb.org/about/#maintenance-policy) at the foundation website.
Also we could include the link to [Platform Deprecation Policy](https://mariadb.com/kb/en/mariadb-platform-deprecation-policy/) for studying the policices further for those who might express extra interest.

Reply:
I actually really want us to have something similar to endoflife.date (in fact I wanted to go futher), but I failed to convince people. Probably best it stays some kind of MariaDB Foundation / Plc owned link.

Reply:
@LinuxJedi I can't see how would this be possibly done, by explicitly specifying a list in the committed file and without the maintainance issue.

The best I'd suggest is for example to embed a file f/ex named DEPRECATED in the repository root.
Then, a script invoking a few git commands can be written in the documentation (iterating through the `1.\..?` named branches). Still only a hack. 

Perhaps, otherwise, a single file can be put in a mariadb ftp directory.

Reply:
I agree this is definitely not ideal as it would require maintenance on several branches.
We could simply replace the recommendation with a link that is maintained externally
* New feature target: https://github.com/MariaDB/server/blob/-/VERSION
* Earliest maintained version: https://mariadb.org/about/#maintenance-policy